File: frontend/src/app/models/Cliente.ts
export interface Cliente {
	nitCliente: string;
	nombresCliente: string;
	apellidosCliente: string;
	telefono: string;
	correo: string;
	direccionEntrega: string;
	noTarjeta: string;
	nacionalidad: string;
}

export interface FormCliente {
	correo: string;
	password: string;
}

export interface RegistroCliente {
	nitCliente: string;
	nombresCliente: string;
	apellidosCliente: string;
	telefono: string;
	correo: string;
	direccionEntrega: string;
	noTarjeta: string;
	password: string;
}

File: backend/Domain/Cliente.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Identity;

namespace Domain
{
    public class Cliente : IdentityUser
    {
        public int NitCliente { get; set; }
        public string NombresCliente { get; set; }
        public string ApellidosCliente { get; set; }
        public string DireccionEntrega { get; set; }
        public Pais PaisResidencia { get; set; } //Creates foreign key
        public ICollection<Tarjeta> Tarjetas { get; set; } //Creates foreign key on oposite table N:1
    }
}